contracts/p1/mixins/RecollateralizationLib.sol

```diff
@@ -329,26 +282,33 @@ library RecollateralizationLibP1 {
         Registry memory reg,
         BasketRange memory range
     ) private view returns (TradeInfo memory trade) {
+        // assert(tradesOpen == 0); // all balances in ctx are guaranteed to be available
```

Comment:
I'm down to actually assert it

Reply:
Oh right, it's because `tradesOpen` is in `BackingManager` and we're in the recollat library here. There's a require in `BackingManager.rebalance()` that checks this.